### PR TITLE
Add cell cut

### PR DESCRIPTION
**Pull request by captainsafia**
It's time for Safia to get out of her coding funk.

I heard you like to cut cells too.

cut-paste

Also, I should probably change paste so that it pastes after the focused cell.

### CODE REVIEW

File: src/notebook/actions.js [hunk 1]
```diff
@@ -231,6 +231,13 @@ export function copyCell(id) {
   };
 }
 
+export function cutCell(id) {
+  return {
+    type: constants.CUT_CELL,
+    id,
+  };
+}
```
Comment (rgbkrk) [inline src/notebook/actions.js:239]: Testing the action creator will increase coverage in a pretty simple way here (pretty much a copy paste of other action creator tests).

Comment (captainsafia) [reply src/notebook/actions.js:239]: Ah. Whoops. Forgot about action creators tests.

Coming right up!

File: src/notebook/components/notebook.js [hunk 2]
```diff
@@ -148,6 +154,8 @@ class Notebook extends React.Component {
       const cmdOrCtrl = e.ctrlKey || e.metaKey;
       if (cmdOrCtrl && e.shiftKey && e.keyCode === 67) {
         this.copyCell();
+      } else if (cmdOrCtrl && e.shiftKey && e.keyCode === 88) {
+        this.cutCell();
```
Comment (rgbkrk) [inline src/notebook/components/notebook.js:158]: If you use a similar test to the Markdown cell key handling, you can check to make sure that this function gets called.

File: src/notebook/components/notebook.js [hunk 3]
```diff
@@ -138,6 +140,10 @@ class Notebook extends React.Component {
     this.props.dispatch(copyCell(this.props.focusedCell));
   }
 
+  cutCell() {
+    this.props.dispatch(cutCell(this.props.focusedCell));
```
Comment (rgbkrk) [inline src/notebook/components/notebook.js:144]: This can be mocked with a fake dispatch to ensure it generates the right action.

**Comment by rgbkrk** [comment]
Added some notes about how to test the other new bits being introduced here.

**Comment by captainsafia** [comment]
Updated. I've got another PR going with some component tests so I'll address the component-related issues there.

**Comment by rgbkrk** [comment]
Woot!

**Pull request merged**